documentation/gitbook/how-to-run/advanced-run-guides/quick-start-guide-conda.md

```diff
@@ -153,19 +131,19 @@ The next step depends on what sort of simulation you want to run: One that inclu
 In either case, navigate to the project folder and make sure to delete any old model output files that are there.
 
 ```bash
-cd $DATA_PATH       # goes to your project repository
+cd $PROJECT_PATH       # goes to your project repository
 rm -r model_output/ # delete the outputs of past run if there are
 ```
 
 #### Inference run
 
-An inference run requires a configuration file that has an `inference` section. Stay in the `$DATA_PATH` folder, and run the inference script, providing the name of the configuration file you want to run (ex. `config.yml`). In the example data folder (flepimop\_sample), try out the example config XXX.
+An inference run requires a configuration file that has an `inference` section. Stay in the `$PROJECT_PATH` folder, and run the inference script, providing the name of the configuration file you want to run (ex. `config.yml`). 
```

Comment:
the inference action, right? it's a script under the hood, but not from the user perspective